src/runtime/slot-dom-consumer.ts

```diff
@@ -294,18 +295,18 @@ export class SlotDomConsumer extends SlotConsumer {
       }
       const subId = this.getNodeValue(innerContainer, 'subid');
       assert(Boolean(subId) === providedContext.spec.isSet,
-        `Sub-id ${subId} for slot ${providedContext.name} doesn't match set spec: ${providedContext.spec.isSet}`);
+             `Sub-id ${subId} for slot ${providedContext.name}/${slotId} doesn't match set spec: ${providedContext.spec.isSet}` + innerContainer);
```

Comment:
If you are changing this, suggest you change it to say something like `Slot .... is missing sub-id`. The `... doesn't match set spec: true` output is confusing.

Reply:
done